hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileChecksum.java

```diff
@@ -68,10 +71,16 @@ public class TestOzoneFileChecksum {
       true, false
   };
 
-  private static final int[] DATA_SIZES = DoubleStream.of(0.5, 1, 1.5, 2, 7, 8)
-      .mapToInt(mb -> (int) (1024 * 1024 * mb))
+  private static final int[] DATA_SIZES_1 = DoubleStream.of(0.5, 1, 1.5, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8, 9, 10)
```

Comment:
In these test, do we somehow vary the bytes per checksum too? The original issue was hidden as the bytes per checksum == chunk_size. So it would be good to have a test that covers that.

Reply:
Yes, I can add that as well. Right now it's `16KB`